packages/components/src/card/card/component.js

```diff
@@ -11,42 +11,59 @@ import { useMemo } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { contextConnect } from '../context';
+import { contextConnect, ContextSystemProvider } from '../../ui/context';
 import { Elevation } from '../../elevation';
 import { View } from '../../view';
-import * as styles from './styles';
+import * as styles from '../styles';
 import { useCard } from './hook';
 import CONFIG from '../../utils/config-values';
 
 /**
- * @param {import('../context').PolymorphicComponentProps<import('./types').CardProps, 'div'>} props
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').Props, 'div'>} props
  * @param {import('react').Ref<any>} forwardedRef
  */
 function Card( props, forwardedRef ) {
-	const { children, elevation, isRounded = true, ...otherProps } = useCard(
-		props
-	);
+	const {
+		children,
+		elevation,
+		isBorderless,
+		isRounded,
+		size,
+		...otherProps
+	} = useCard( props );
 	const elevationBorderRadius = isRounded ? CONFIG.cardBorderRadius : 0;
 
 	const elevationClassName = useMemo(
 		() => css( { borderRadius: elevationBorderRadius } ),
 		[ elevationBorderRadius ]
 	);
 
+	const contexProps = {
+		size,
+		isBorderless,
+	};
+	const contextProviderValue = {
+		CardBody: contexProps,
+		CardHeader: contexProps,
+		CardFooter: contexProps,
+	};
+
```

Comment:
This should be memoized using `useMemo`

Reply:
Well spotted! Done in https://github.com/WordPress/gutenberg/pull/32243/commits/dba8d98278d066c67de172d4dcab060b41c45e39